docs/10-faq/index.mdx

```diff
@@ -0,0 +1,262 @@
+---
+title: FAQ
+---
```

Comment:
Everywhere else we do this using a [\_category_.yaml](https://github.com/game-ci/documentation/blob/main/docs/09-troubleshooting/_category_.yaml) file, which gives more flexibility.

Reply:
Fixed, I updated the title directly so we don't have weird title levels. `Frequently Asked Questions (FAQ)`.